scripts/oldump.sh

```diff
@@ -103,8 +103,8 @@ function archive_dumps() {
     # copy stuff to archive.org
     # TODO: Switch to ia client tool. This will only work in production 'til then
     log "ia version is v$(ia --version)"  # ia version is v1.9.0
```

Comment:
From https://pypi.org/project/internetarchive/
```suggestion
    log "ia version is v$(ia --version)"  # ia version is v2.2.0
```